CONTRIBUTING.md

```diff
@@ -690,14 +743,17 @@ designated as a development coordinator, and has to approve the
 changes in the core team membership (until we get a more formal
 joining and leaving process).
 
-The core developers are the members of the **@coq/core** team.
+The core developers are the members of the **@coq/core** team ([member
+list][coq-core] only visible to the Coq organization members because
+of a limitation of GitHub).
```

Comment:
How does one join the organization?  Can anyone join or is it an exclusive club :-)?

Reply:
I'm not sure to understand exactly what you are asking. Are you saying that this link is not visible to you? Or that the document should give the list of core members to people who are not in the org (yes, there should be a webpage with this list, but this is not the purpose of this PR to make it happen). The triaging bug section already mentions joining the contributors team.

Reply:
Just clarify what "Coq organization members" means.  That's not made clear elsewhere in the doc.  I suppose it means being a member of  @coq/contributors?  Or just say "visible to members of @coq/contributors".

Reply:
Please use `@coq/contributors` (in backtics) unless you actually want to ping every member of that team. You just subscribed 48 people to this PR, that will all have to individually manually unsubscribe. ;)

Reply:
Coq organization members are the people who are members of the Coq organization on GitHub. It may seem like a tautological statement but I don't know how you can make this more precise. It is not the same as the contributors team because some people were added as members of this organization without being added to the contributors team (and thus do not have write-access to the Coq repository). This is the case, in particular, of the people who were added to the `@coq/opam-maintainer` team. This parenthesis is only about documenting a *technical limitation* of GitHub because you asked me to add a link to the member list. It is not about a policy. If this is too confusing, I should just remove the parenthesis. It's not like these links are very important anyway.